app/assets/javascripts/battletype/ship.js

```diff
@@ -26,25 +48,31 @@
         get: function () { return this._targeted; },
         set: function (t) {
           this._targeted = Boolean(t);
-          
+
           var s = this.querySelector(".ship");
           if (this._targeted) {
             s.classList.add("targeted_ship");
           } else if (s.classList.contains("targeted_ship")) {
             s.classList.remove("targeted_ship");
           }
-          
+
           return this._targeted;
         }
+      },
+      type: {
```

Comment:
Same as above. For now I couldn't find a way to evaluate the height of the ship. Any ideas? :)